protocol/parser/parser.go

```diff
@@ -275,9 +287,11 @@ type ParsedInput struct {
 	parsedHashes  []string
 }
 
+const RAW_NOT_APPLICABLE = "-1"
+
 func NewParsedInput() *ParsedInput {
 	return &ParsedInput{
```

Comment:
you didn't fix my comment here, add the parsed block with a default value of not applicable, the default value of 0 is wrong as it can be a valid block

Reply:
Fixed in 722362c25e9ed43f272989461697a7cc5d4ca993